Tools/autotest/common.py

```diff
@@ -9062,20 +9100,21 @@ def test_frsky_sport(self):
         # This, at least makes sure we're getting some of each
         # message.
         wants = {
-            0x01:  self.tfs_validate_gps_alt, # gps altitude integer m
-            0x02:  self.tfs_validate_tmp1, # Tmp1
-            0x04:  self.tfs_validate_fuel, # fuel
-            0x05:  self.tfs_validate_tmp2, # Tmp2
-            0x09:  lambda x : True, # gps altitude decimal cm
-            0x10:  self.tfs_validate_baro_alt, # baro alt integer m
-            0x11:  self.tfs_validate_gps_speed, # gps speed integer m/s
-            0x14:  self.tfs_validate_yaw, # yaw in degrees
-            0x19:  lambda x : True, # gps speed decimal cm/s
-            0x21:  lambda x : True, # altitude decimal m
-            0x28:  self.tfs_validate_current1, # current A
-            0x30:  self.tfs_validate_vspeed, # vertical speed m/s
-            0x39:  self.tfs_validate_battery1, # battery 1 voltage
-            0x800: self.tf_validate_gps, # gps lat/lon
+            0x01:   self.tfs_validate_gps_alt, # gps altitude integer m
```

Comment:
Could have saved a lot of delta here by not having a leading zero in your new key :-)

Reply:
haha, oops, I'll fix it :-)